qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -1241,6 +1241,77 @@ def test_can_be_updated_on_new(self):
                                  self.data_type)
         self.assertTrue(pt.can_be_updated({'barcode'}))
 
+    def test_extend_add_samples(self):
```

Comment:
Could you also add a test for update/extend at the same time? I couldn't find it in the code.